src/modules/utility/commands/presences.ts

```diff
@@ -55,6 +55,17 @@ export default new DiscordCommand({
 
 		if (!dbPresence) return;
 
+		if (dbPresence.metadata.tags.includes("nsfw") && (int.channel?.type == "GUILD_TEXT" && !int.channel.nsfw)) {
+			await int.editReply({
+				embeds: [{
+					title: ":mag_right: Presences • Error",
```

Comment:
Title seems very strange as it’s not really an error

Reply:
Oops, my bad, I copied it from older commit, what should I change it to?